# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xfree86/ftconfig-h-hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- xc/lib/font/FreeType/ftconfig.h~	2003-11-21 06:57:51.000000000 +0100
+++ xc/lib/font/FreeType/ftconfig.h	2003-12-21 23:03:36.000000000 +0100
@@ -63,7 +63,7 @@
 
 #ifdef XFREE86_FT2
 
-# include "X11/Xmd.h"
+/* # include "X11/Xmd.h" */
 
   /* The number of bytes in an `int' type.  */
 # define FT_SIZEOF_INT 4
